spread-shellcheck

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
```

Comment:
Is `set -e` missing in this script or am I just not looking carefully enough?

Reply:
Added